modules/evm/src/runner/native.rs

```diff
@@ -53,14 +53,27 @@ impl<T: Trait> RunnerT<T> for Runner<T> {
 		let mut substate =
 			Handler::<T>::new_with_precompile(&vicinity, gas_limit as usize, false, config, T::Precompiles::execute);
 
+		if !value.is_zero() {
+			substate
+				.transfer(Transfer { source, target, value })
+				.map_err(|_| Error::BalanceLow)?;
+		}
+
+		substate
+			.withdraw(source, U256::from(gas_limit))
```

Comment:
we don't charge gas. the standard tx fee already handles it

Reply:
I was trying to follow StackRunner. Anyway will address it later if needed